registry/infrastructure/repositories/organization_repository.py

```diff
@@ -153,14 +154,14 @@ def add_organization(self, organization, username, state, address="", test_trans
         ))
 
         self.add_item(OrganizationMember(
-            invite_code=uuid4().hex, org_uuid=organization.uuid, role=Role.OWNER.value, username=username, address=address,
-            status=OrganizationMemberStatus.ACCEPTED.value, transaction_hash="", invited_on=current_time,
-            created_on=current_time, updated_on=current_time))
+            invite_code=uuid4().hex, org_uuid=organization.uuid, role=Role.OWNER.value, username=username,
+            address=address, status=OrganizationMemberStatus.ACCEPTED.value, transaction_hash="",
+            invited_on=current_time, created_on=current_time, updated_on=current_time))
 
     def _update_organization(self, organization_db_model, organization, username, state, test_transaction_hash=None):
         current_time = datetime.utcnow()
         organization_db_model.name = organization.name
-        organization_db_model.id = organization.id
+        organization_db_model.org_id = organization.id
```

Comment:
Id in the domain model and org_id in DB model. Why?

Reply:
No specific reason. kept it as is from the metadata